backend/attachRendererFiber.js

```diff
@@ -239,8 +250,8 @@ function attachRendererFiber(hook: Hook, rid: string, renderer: ReactRenderer):
         const inst = publicInstance;
         if (inst) {
           updater = {
+            canUpdate: true,
             setState: inst.setState && inst.setState.bind(inst),
-            forceUpdate: inst.forceUpdate && inst.forceUpdate.bind(inst),
             setInProps: inst.forceUpdate && setInProps.bind(null, fiber),
```

Comment:
Would not overriding `setInProps` if `updater && updater.setInProps` fix https://github.com/facebook/react-devtools/issues/212 by using `scheduleWork` instead of `forceUpdate`?

Reply:
Try and see?